slack_sdk/web/client.py

```diff
@@ -119,32 +114,114 @@ def admin_apps_approve(
                 "The app_id or request_id argument must be specified."
             )
 
-        return self.api_call("admin.apps.approve", json=kwargs)
+        kwargs.update(
```

Comment:
when we use params (not json), `None` values will be removed in the `api_call` method (both async and sync)